tests/Aspire.Hosting.Tests/WithHttpCommandTests.cs

```diff
@@ -325,6 +325,8 @@ public async Task WithHttpCommand_EnablesCommandOnceResourceIsRunning()
         // Arrange
         using var builder = TestDistributedApplicationBuilder.Create(testOutputHelper);
 
+        builder.Configuration["CODESPACES"] = "false";
```

Comment:
Why is this necessary?

Reply:
When running this test in a Codespace the URLs will be remapped in a way that this test doesn't expect. May need to verify that the command enable/disable behavior doesn't try to use the codespaces URL and instead uses it from the endpoint annotation.

Reply:
/cc @DamianEdwards

Reply:
https://github.com/dotnet/aspire/issues/8070